sunpy/net/helio/hec.py

```diff
@@ -156,39 +156,29 @@ def make_table_list(self):
 
         Returns
         -------
-        temp: `str`
+        `bytes`
```

Comment:
Should this be bytes and not a string?

Reply:
Yes.  Method `make_table_list` returns a `bytes` object, not `str`.

Reply:
@nabobalis I want to mention that this service will work if `table` is either string or bytes object. But it would be best to use bytes because that how client and TimeQuery service parses them. 
```python3
TimeQuery(STARTTIME=start_time.isot, ENDTIME=end_time.isot, FROM=table, MAXRECORDS=max_records)
```

Reply:
Yeah that is fine.